core/src/main/scala/cromwell/core/simpleton/WdlValueBuilder.scala

```diff
@@ -0,0 +1,145 @@
+package cromwell.core.simpleton
+
+import wdl4s.TaskOutput
+import wdl4s.types.{WdlArrayType, WdlMapType, WdlPrimitiveType, WdlType}
+import wdl4s.values.{WdlArray, WdlMap, WdlValue}
+
+import scala.language.postfixOps
+import cromwell.core.simpleton.WdlValueSimpleton._
+
+
+/** Builds arbitrary WdlValues from WdlValueSimpletons. */
+object WdlValueBuilder {
+
+  private val SimpletonPattern = """
```

Comment:
I had the reverse comment on the regexp I copied this from.  😄 

Though TBH the first two are probably simple enough I would agree with that; I'm less sure about the third.